test/binremote/binremote/archiver.go

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Can you flatten all these files down one directory?  ie, get rid of the second binremote